README.adoc

```diff
@@ -68,29 +69,34 @@ mvn install liberty:start-server
 Maven builds the application and runs it inside of Open Liberty.
 
 Point your browser to the `http://localhost:9080/inventory/systems/` URL. Because you recently started the application,
-hosts are not currently registered. Point to the `http://localhost:9080/inventory/systems/localhost` URL to add a host. Next, Point to the
-`https://localhost:9443/metrics` MicroProfile Metrics endpoint. Click the `Advanced` button to
+hosts are not currently registered. Point to the `http://localhost:9080/inventory/systems/localhost` URL to add a host.
+
+Next, point to the `https://localhost:9443/metrics` MicroProfile Metrics endpoint. Click the `Advanced` button to
 proceed. Use the `confAdmin` and `microprofile` credentials to log in. You can see both the standard metrics and
-the application metrics in Prometheus format. To see only the application metrics, point your browser to
+the application metrics in Prometheus format.
```

Comment:
By the way, why Prometheus and not JSON, which is also supported?

Reply:
@gkertaseferi had it Prometheus previously, so I left it as it is. Also /metrics does not show JSON. So thats why I think she used "Prometheus" word.

Reply:
It does if we request for application/json per spec.  @gkertaseferi, any thoughts to add?

Reply:
Yes It does, but i meant for the current guide it does not show json atm

Reply:
Yes, hence my question on why.  Thanks.